pages/login.php

```diff
@@ -0,0 +1,54 @@
+<?php
+# Licensed under the MIT license
+
+require_once( 'core.php' );
+require_api( 'authentication_api.php' );
+require_api( 'user_api.php' );
+
+$f_username = gpc_get_string( 'username' );
+$f_reauthenticate = gpc_get_bool( 'reauthenticate', false );
+$f_return = gpc_get_string( 'return', config_get( 'default_home_page' ) );
+
+$t_return = string_url( string_sanitize_url( $f_return ) );
+
+require_once( plugin_config_get( 'autoloader_path' ) );
+$t_simplesamlphp_instance = new SimpleSAML_Auth_Simple( plugin_config_get( 'SP_name' ) );
+$t_simplesamlphp_instance->requireAuth();
+if( $t_simplesamlphp_instance->isAuthenticated() ) {
+	$t_simplesamlphp_attributes = $t_simplesamlphp_instance->getAttributes();
+    $f_username = $t_simplesamlphp_attributes[ plugin_config_get( 'auth_attributes_username' ) ][0];

```

Comment:
Check that the attribute is set before retrieving it to avoid php errors in case plugin is misconfigured during testing.